### DISCUSSION TITLE
textarea at supabase/ui

### DISCUSSION THREAD

**Discussion by florian-lefebvre** [supabase/supabase#939] [Feature Requests] [answered]
Looking at https://ui.supabase.com/?path=/docs/data-input-input--text-area, I think that only the height should be editable and not the width. It would be also great to put a max-height and a min-height. I didn't know where to report so I hope it's the right place for that.

**Comment by silentworks** [comment, marked as answer]
This is repo is probably the right place, but open it as an issue instead.

**Reply by florian-lefebvre** [reply]
thx!
